relnotes/turtleenv.migration.md

```diff
@@ -0,0 +1,20 @@
+## Turtle Node ext base class is now in swarm
+
+`turtle.env.model.Node`
+
+This module providing the common turtle ext node base class used to be included
+in the protos. Because it's the same file, it's now moved into swarm. Proto
+repositories should remove their copy of this file.
+
+Couple of changes needs to be performed while updating the protos' implementations,
+and client code:
+
+- `ignoreErrors()` is now renamed to `logErrors(bool)` method which can
+  enable/disable the error output.
+
+- `node_item()` method has renamed to `node_addport` and now it returns
+  `AddrPort` structure. Make sure you don't set the port/address directly,
```

Comment:
>Make sure you don't set the port/address directly

What does this refer to? The addr/port is returned by value from `node_addrport`...

Reply:
It's referring to this bit, and it's only relevant for the implementations of this: https://github.com/sociomantic-tsunami/swarm/pull/232/files#diff-f5dd25daf769535b13e67e8f028451e3R31

If we were about to construct the `AddrPort` directly instead:

```
this.neo_address = AddrPort(this.addrport.address(), this.addport.port() + 100)
```

which was the usual case with the `NodeItem` structure, this would result in a problem, since the `AddrPort` stores the fields in the network byte order, and `port()+100` returns it in host byte order.

Reply:
Hmm maybe give a before / after?